### PR TITLE
refac encode

### PR DESCRIPTION
**Pull request by AgwaB**
details:
Refactoring the encoding/encode

- [x] Test case

### CODE REVIEW

**Review by junbeomlee — APPROVED** [review]
LGTM!

File: encoding/encode.go [hunk 1]
```diff
@@ -17,24 +17,14 @@
 package encoding
 
 import (
+	"encoding/binary"
 	"encoding/hex"
 	"fmt"
-	"strconv"
 
 	"github.com/pkg/errors"
```
Comment (hihiboss) [inline encoding/encode.go]: Please change `"github.com/pkg/errors"` to `"errors"`.

File: encoding/encode.go [hunk 2]
```diff
@@ -44,61 +34,40 @@ func (e EncodeError) Error() string {
...
+func encodeInt(operand int64) ([]byte, error) {
+	byteSlice := make([]byte, 8)
+	binary.BigEndian.PutUint64(byteSlice, uint64(operand))
 
-	return b, nil
+	return byteSlice, nil
 }
 
 // Encode string to hexadecimal bytes
 // ex) string "abc" => 0x616263
```
Comment (hihiboss) [inline encoding/encode.go]: Please fix the comment like `0x6162630000000000`.

**Review by hihiboss — CHANGES_REQUESTED** [review]
Fix them!

**Review by hihiboss — APPROVED** [review]

**Pull request merged**